house/views.py

```diff
@@ -94,4 +94,25 @@ def house_create(request):
     else:
         house_form = HouseCreationForm()
 
-    return render(request, HOUSE_CREATE_ROUTE, {'form': house_form, 'msg': errorMsg})
+    return render(request, HOUSE_CREATE_ROUTE, {'form': house_form, 'msg': error_message})
+
+
+@login_required
+def edit_expense(request, id):
+    expense = get_object_or_404(Expenses, pk=id)
```

Comment:
Please add a condition that make sure that the expenses is really on of the user expenses

Reply:
fixed